urdf/insta360_air.xacro

```diff
@@ -1,41 +1,50 @@
 <?xml version="1.0"?>
 
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-    <xacro:property name="M_PI" value="3.14159"/> 
+    <xacro:property name="M_PI" value="3.14159"/>
+    
+    <!-- TODO: check datasheet -->
+    <xacro:property name="insta360_air_mass" value="0.05"/>
+    <xacro:property name="insta360_air_length" value="0.021"/>
+    <xacro:property name="insta360_air_width" value="0.07"/>
+    <xacro:property name="insta360_air_height" value="0.025"/>
 
-    <xacro:macro name="insta360_air" params="name:=default *origin *parent">
-        <joint name="insta360_air_${name}_base_joint" type="fixed">
-            <child link="insta360_air_${name}_base_link"/>
+    <xacro:include filename="$(find hector_xacro_tools)/urdf/inertia_tensors.urdf.xacro" />
+
+    <xacro:macro name="${name}" params="name:=default *origin *parent"> 
+        <joint name="${name}_joint" type="fixed">
+            <child link="${name}_link"/>
             <xacro:insert_block name="origin"/>
             <xacro:insert_block name="parent"/>
         </joint>
 
         <!-- Camera base -->
-        <link name="insta360_air_${name}_base_link">
+        <link name="${name}_link">
+            <xacro:inertial_cuboid mass="${insta360_air_mass}" x_length="${insta360_air_length}" y_length="${insta360_air_width}" z_length="${insta360_air_height}" />
             <visual>
                 <origin xyz="0 0 0" rpy="0 0 0" />
                 <geometry>
-                    <box size="0.021 0.07 0.025"/>
+                    <box size="${insta360_air_length} ${insta360_air_width} ${insta360_air_height}"/>
                 </geometry>
             </visual>
             <collision>
                 <origin xyz="0 0 0" rpy="0 0 0" />
                 <geometry>
-                    <box size="0.021 0.07 0.025"/>
+                    <box size="${insta360_air_length} ${insta360_air_width} ${insta360_air_height}"/>
                 </geometry>
             </collision>
         </link>
 
         <!-- Camera RGB sensor -->
-        <link name="insta360_air_${name}_left_optical_frame"/>
-        <joint name="insta360_air_${name}_left_optical_joint" type="fixed">
+        <link name="${name}_rgb_left"/>
```

Comment:
Why did you remove the left optical frame?  There is also no need to prefix "rgb", because this is just a color camera.